pyanaconda/install.py

```diff
@@ -107,6 +108,12 @@ def doConfiguration(storage, payload, ksdata, instClass):
     with progress_report(_("Generating initramfs")):
         payload.recreateInitrds()
 
+    # Work around rhbz#1200539, grubby doesn't handle grub2 missing initrd with /boot on btrfs
+    # So rerun writing the bootloader if this is live and /boot is on btrfs
+    if flags.flags.livecdInstall and (not ksdata.bootloader.disabled and ksdata.bootloader != "none") \
+       and isinstance(storage.mountpoints.get("/boot", storage.mountpoints.get("/")), BTRFSDevice):
```

Comment:
Would you mind moving this part ``isinstance(storage.mountpoints.get("/boot", storage.mountpoints.get("/")), BTRFSDevice)`` into a separate variable like ``boot_on_btrfs`` assigned before the if and used in the long/complex condition? I'd find that better readable.